machine/v1beta1/types_gcpprovider.go

```diff
@@ -54,9 +54,20 @@ type GCPMachineProviderSpec struct {
 	// ProjectID is the project in which the GCP machine provider will create the VM.
 	// +optional
 	ProjectID string `json:"projectID,omitempty"`
+	// GuestAccelerators is a list of GPUs to be attached to the VM.
+	// +optional
+	GuestAccelerators []*GCPAcceleratorConfig `json:"guestAccelerators,omitempty"`
 	// Preemptible indicates if created instance is preemptible
 	// +optional
 	Preemptible bool `json:"preemptible,omitempty"`
+	// OnHostMaintenance Determines the behavior when a maintenance event occurs that might cause the instance to reboot (default MIGRATE).
+	// This is required to be set to TERMINATE if you want to provision machine with attached GPUs.
+	// +optional
+	OnHostMaintenance string `json:"onHostMaintenance,omitempty"`
+	// AutomaticRestart Determines the behavior when an instance crashes or is stopped by the system (default true).
+	// Can not be true with preepmtible instances.
```

Comment:
Does "the system" mean OpenShift, GCP, or something else?  
```suggestion
	// AutomaticRestart determines the behavior when an instance crashes or is stopped by the system (default true).
	// Cannot be true with preemptible instances.
```

Reply:
This is taken from GCP docs, `instance` is referring to the VM running on GCP. 
If it seems confusing i can update this with clearer description.